roles/robottelo/tasks/ssh.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: 'Check if SSH key exists'
+  stat:
+    path: "{{ robottelo_ssh_key }}"
+  register: st
+
+- name: 'Create ssh keys'
+  command: "ssh-keygen -b 2048 -t rsa -f {{ robottelo_ssh_key }} -q -N ''"
+  when: st.stat.exists == false
+
+- name: 'Add ssh key to authorized_keys'
```

Comment:
Won't this break vagrant SSH login or will that continue to work because that's the `vagrant user?